test/controllers/FileChecksResultsControllerSpec.scala

```diff
@@ -403,8 +403,6 @@ class FileChecksResultsControllerSpec extends FrontEndTestHelper {
     transferAlreadyCompletedPageAsString must include(
       s"""href="/judgment/$consignmentId/transfer-complete">Continue""".stripMargin)
     transferAlreadyCompletedPageAsString must include("Your transfer has already been completed")
-    transferAlreadyCompletedPageAsString must include (s"""" href="/faq">""")
```

Comment:
Yes, in [this PR](https://github.com/nationalarchives/tdr-transfer-frontend/pull/1788/files#) (on line 468 at the very bottom of the page), I'm going to move it so that it is [checked here](https://github.com/nationalarchives/tdr-transfer-frontend/blob/36536f8de041e9c00ba101a7216973b4666c9e54/test/util/CheckPageForStaticElements.scala#L33)